mobify/scooch
# Apply Modern Standards + Add Infinite Option + Add Autoplay

Reviewers: @jansepar @mikenikles @fractaltheory @MikeKlemarewski 

## Changes
Update Scooch to some modern best-practices
- Add `autoplay` option for automatic looping through carousels ( Fixes #5 )
- Add `infinite` option for looping carousels ( Fixes #37 )
- Prep for publishing to NPM
- De-emphasize Zepto in favor of jQuery
- Lint JS source code with eslint
- Update versions of dependencies `grunt-contrib-connect` and `grunt-contrib-uglify`
- Integrate with CircleCI
- Add some GitHub templates for contributing, issues, and PRs

## Todos:
- [x] Github admin to add Scooch repo to CircleCI watching
- [x] +1 (Major version bump)
- [x] Add a PR template
- [x] Better documentation in Readme for Scooch options ( Fixes #34 )
- [x] Add autoplay option ( Fixes #5 )

## How to Test
- Checkout code
- Refresh node modules with `rm -rf node_modules && npm install`
- `grunt lint`
- Check that the examples still work with `grunt examples`
- Check out [this branch of Extra Stores](https://github.com/mobify/extra-mobile/tree/DNM-testing-modern-scooch) that uses this new code for its Scooches

## Applicable Research Resources
- Autoplay code ported from [Extra](https://github.com/mobify/extra-mobile/blob/master/adaptation/utils/utils.js#L69)
